src/foam/core/reflow/PropertyRichListView.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.core.reflow',
+  name: 'PropertyRichListView',
+  extends: 'foam.core.reflow.PropertyListView',
```

Comment:
Why not just do this to PropertyListView? Do we really need both of them?

Reply:
My question is do we need two kinds of property list views? Is there any case we will use PropertyListView as opposed to PropertyRichListVIew. I am proposing just using the RichChoiceView in PropertyListView, and removing the old PropertyListView. And in conjunction with my other comment, removing the PropertyRichChoiceView as well since that can all be handled in view config

Reply:
@ajeetgill ^